tooling/templatize/cmd/pipeline/inspect/cmd.go

```diff
@@ -0,0 +1,42 @@
+package inspect
+
+import (
+	"context"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCommand() *cobra.Command {
+	opts := DefaultOptions()
+	cmd := &cobra.Command{
+		Use:   "inspect",
+		Short: "inspect aspects of a pipeline.yaml file",
+		Long:  "inspect aspects of a pipeline.yaml file",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx, stop := signal.NotifyContext(cmd.Context(), os.Interrupt, syscall.SIGTERM)
+			defer stop()
+
+			return runInspect(ctx, opts)
+		},
+	}
+	if err := BindOptions(opts, cmd); err != nil {
+		log.Fatal(err)
```

Comment:
nit: almost always better to have `os.Exit()` type stuff only in a `func main()` and return errors elsewhere so you can reason about control flow.